README.md

```diff
@@ -15,6 +15,16 @@ Through this interface you should set the following configuration to the ``Polar
 + **writeFullMessageToConsole** (*boolean - optional*) - If you do decide to write logs to console, only the timestamp 
 accompanied by the log level and message will be written, in order to see the whole log, which includes the properties 
 that would be sent to the logstash, set this property to ``true``.
++ **logFilePath** (*string - optional*) - If provided, the logs will be written to the specified path.
++ **dailyRotateFileConfiguration** (*DailyRotateFileConfiguration - optional*) - If you are interested in daily log file
+instead of just **one** log file, see the configuration section below.
+
+### DailyRotateFileConfiguration
```

Comment:
add description - it creates a log file for every day that is deleted after X days